### ISSUE TITLE
Page sizes in preview

### ISSUE THREAD

**Issue by DDDenisSobek** [stsrki/Blazorise#1413]
**Describe the bug**
Changing of paging sizes will not recognize.

**To Reproduce**
Make a page with code like below. Then change page size and nothing happen. Page buttons are also not right.
```xml
<h3>DataGridTestPage</h3>

<DataGrid TItem="TestItem" Data="@Items" ReadData="ReadItems" 
          ShowPager="true" TotalItems="@_AllItems.Count" ShowPageSizes="true">
    <ChildContent>
        <DataGridColumn TItem="TestItem" Field="@nameof(TestItem.Id)" Caption="Id" />
        <DataGridColumn TItem="TestItem" Field="@nameof(TestItem.Name)" Caption="Name" />
    </ChildContent>
</DataGrid>
```
```
@code {

    public class TestItem
    {
        public int Id { get; set; }
        public string Name { get; set; }
    }

    private List<TestItem> _AllItems = new List<TestItem>();

    public List<TestItem> Items { get; set; } = new List<TestItem>();

    protected override void OnInitialized()
    {
        base.OnInitialized();
        for (int i = 0; i < 1000; i++)
        {
            _AllItems.Add(new TestItem()
            {
                Id = i,
                Name = "Test" + i
            });
        }
    }

    private void ReadItems(DataGridReadDataEventArgs<TestItem> args)
    {
        var offset = (args.Page - 1) * args.PageSize;
        Items = _AllItems.Skip(offset).Take(args.PageSize).ToList();
    }
}
```
**Screenshots**
![grafik](https://user-images.githubusercontent.com/65016836/97201889-3cb4d180-17b3-11eb-83ea-02a146309412.png)
![grafik](https://user-images.githubusercontent.com/65016836/97202065-6f5eca00-17b3-11eb-95ea-d305730c2f7b.png)
![grafik](https://user-images.githubusercontent.com/65016836/97202143-856c8a80-17b3-11eb-8615-7efb732c55af.png)

**Comment by DDDenisSobek** [comment]
Hey, does something exist new?

**Comment by stsrki** [comment]
@DDDenisSobek This should be fixed in 0.9.2-rc3. There was an issue with TotalItems not being applied properly. Check PR #1446

I will close this, and if the problem is still there, please reopen.

**Comment by DDDenisSobek** [comment]
Sorry for late response.
I have tested it with version 0.9.2.1, but the bug still exists.
